cypress/integration/pages/mostReadPage/index.js

```diff
@@ -1,31 +1,54 @@
-import runTestsForPage from '../../../support/helpers/runTestsForPage';
-import {
-  testsThatAlwaysRun,
-  testsThatFollowSmokeTestConfig,
-  testsThatNeverRunDuringSmokeTesting,
-} from './tests';
-import {
-  testsThatAlwaysRunForAMPOnly,
-  testsThatFollowSmokeTestConfigForAMPOnly,
-  testsThatNeverRunDuringSmokeTestingForAMPOnly,
-} from './testsForAMPOnly';
-import {
-  testsThatAlwaysRunForCanonicalOnly,
-  testsThatFollowSmokeTestConfigForCanonicalOnly,
-  testsThatNeverRunDuringSmokeTestingForCanonicalOnly,
-} from './testsForCanonicalOnly';
+import config from '../../../support/config/services';
+import getPaths from '../../../support/helpers/getPaths';
+import serviceHasPageType from '../../../support/helpers/serviceHasPageType';
+import testsForCanonicalOnly from './testsForCanonicalOnly';
+import crossPlatformTests from './tests';
+import visitPage from '../../../support/helpers/visitPage';
 
-const testsForPage = {
-  pageType: 'mostReadPage',
-  testsThatAlwaysRun,
-  testsThatAlwaysRunForCanonicalOnly,
-  testsThatAlwaysRunForAMPOnly,
-  testsThatFollowSmokeTestConfig,
-  testsThatFollowSmokeTestConfigForCanonicalOnly,
-  testsThatFollowSmokeTestConfigForAMPOnly,
-  testsThatNeverRunDuringSmokeTesting,
-  testsThatNeverRunDuringSmokeTestingForCanonicalOnly,
-  testsThatNeverRunDuringSmokeTestingForAMPOnly,
-};
-
-runTestsForPage(testsForPage);
+const pageType = 'mostReadPage';
+Object.keys(config)
+  .filter(service => serviceHasPageType(service, pageType))
+  .forEach(serviceId => {
+    // eslint-disable-next-line prefer-const
+    let { variant, name: service } = config[serviceId];
+    if (variant !== 'default') {
+      const capitaliseVariant =
```

Comment:
sounds like a function name because capitalise is verb-y. should maybe be `capitalisedVariant`

Reply:
Done :)